uiowa/uiowa
# Accordion accessibility

Relates to https://github.com/uiowa/uiowa/issues/5106
Added accessible label to accordion header to make sure that it was properly added into the aria mapping. Ran lighthouse, increased accessibility score.

## To test

- sync down sandbox
- go to this page https://sandbox.uiowa.ddev.site/really-cool-accordions
- run a lighthouse scan (This may only work in chrome??) (inspect > click the arrows in the row with the elements tab > Lighthouse > Generate report)
- under the accessibility section check that we are not getting dinged for `Make sure the element is contained by another element that has the required context role.`
- PROFIT
